server/database/queries/cohort/getCohorts.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../../config/connection');
+
+const getCohorts = () => connection.query('SELECT * FROM cohort');
```

Comment:
@AlaaSaadeddin Do you want an `ORDER BY` in this query, to give results to the frontend in a predictable order? Maybe `ORDER BY name` would give a nice result since the names will probably all be "G6", "G7" and so on.

Reply:
Well it is a great idea , I will try to implement and thank you very much  ^_^